src/chrome/content/rules/leboncoin.fr.xml

```diff
@@ -0,0 +1,21 @@
+<ruleset name="leboncoin.fr (Partial)">
```

Comment:
why it's partial? any non-functional subdomain? Please list them as ruleset comment.